interfaces/seccomp/template.go

```diff
@@ -549,3 +549,15 @@ pwritev
 # of socket(), bind(), connect(), etc individually.
 socketcall
 `)
+
+// bindSyscallWorkaround specifies the bind() syscall for systems which
+// don't use AppArmor but only seccomp. In those cases the net package
+// from go uses bind for IPv4/IPv6 support detection when loaded. This
+// gets a problem for all hooks when they use snapctl. On systems which
+// use AppArmor the bind() call is properly mediated and the caller gets
+// an error back.
```

Comment:
Not sure I understand the problem given this description. If we get an error when apparmor is enabled, that would mean the bind always fails. If that's the case, why are systems without apparmor special?

Also, can we please stop using the term "force devmode" in that context?  This sounds like a misnomer for "no apparmor support". We should slowly move all terminology towards that, both in our use and in the code. Some of the misleading conversations and proposals we had recently around that seems to come from that (for example, the idea of disabling seccomp improperly).